src/components/BoardReactionMenu/BoardReactionMenu.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import {ReactComponent as CloseIcon} from "assets/icon-close.svg";
+import {ReactionType} from "types/reaction";
+import {BOARD_REACTION_EMOJI_MAP} from "types/boardReaction";
+import {Actions} from "store/action";
+import {useDispatch} from "react-redux";
+import {useHotkeys} from "react-hotkeys-hook";
+import {useAppSelector} from "store";
+import {Toast} from "utils/Toast";
+import {useTranslation} from "react-i18next";
+import classNames from "classnames";
+import {useDelayedReset} from "utils/hooks/useDelayedReset";
+import "./BoardReactionMenu.scss";
+
+type BoardReactionMenuProps = {
+  close: () => void;
+};
+
+const REACTION_DEBOUNCE_TIME = 300; // milliseconds
+
+export const BoardReactionMenu = (props: BoardReactionMenuProps) => {
+  const dispatch = useDispatch();
+  const {t} = useTranslation();
+
+  const boardReactions = [...BOARD_REACTION_EMOJI_MAP];
+
+  const showBoardReactions = useAppSelector((state) => state.view.showBoardReactions);
+
+  const [debounce, resetDebounce] = useDelayedReset<boolean>(false, true, REACTION_DEBOUNCE_TIME);
+
+  const onClickReaction = (e: React.MouseEvent<HTMLButtonElement> | KeyboardEvent, reaction: ReactionType) => {
+    e.stopPropagation();
+    if (!showBoardReactions) {
+      Toast.info({
+        title: t("Toast.boardReactionsDisabled"),
+        buttons: [t("Toast.enable")],
+        firstButtonOnClick: () => dispatch(Actions.setShowBoardReactions(true)),
+      });
+    } else if (debounce) {
+      // not ready yet
```

Comment:
huh, I think when I wrote this originally it was 
```typescript
else if (debounce) {
    return; // not ready yet
}
```

but the return was removed by the linter on commit automatically, because it's unnecessary syntactically 🙃

Reply:
i tried to add it without hooks but then the linter will fail 🤷